librz/include/rz_magic.h

```diff
@@ -5,6 +5,13 @@
 #define RZ_MAGIC_H
 
 #include <rz_types.h>
+#include <sys/types.h>
+#include <rz_util/rz_rbtree.h>
+#include <sys/queue.h>
```

Comment:
Could this use Rizin's queue implementation instead?
cc @wargio

Reply:
@ahmed-kamal2004 See the `rz_th_queue_..` functions in `rz_th.h`.

Reply:
Not sure if it is the same.